src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -68,6 +68,13 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
    * @type {Array<any>}
    */
   @property() public selectedPeople: Array<MicrosoftGraph.User | MicrosoftGraph.Person | MicrosoftGraph.Contact> = [];
+  /**
+   * accessibile property for user defined selected people
+   *
+   * @type {[string]}
+   * @memberof MgtPeoplePicker
+   */
+  @property() public choosePeople: [];
```

Comment:
Instead of creating a new property, can we use `selectedPeople`? So if I developer simply adds people to the array, it updates the UI?

Reply:
This property should be removed

Reply:
b39cb62